src/app-layout/visual-refresh-toolbar/drawer/global-drawer.tsx

```diff
@@ -75,7 +75,7 @@ function AppLayoutGlobalDrawerImplementation({
     (wasExpanded && !isExpanded);
 
   return (
-    <Transition nodeRef={drawerRef} in={show || isExpanded} appear={show || isExpanded} timeout={0}>
+    <Transition nodeRef={drawerRef} in={show || isExpanded} appear={show || isExpanded} timeout={60}>
```

Comment:
@georgylobko suggested having [unmountOnExit](https://reactcommunity.org/react-transition-group/transition#Transition-prop-unmountOnExit) while keeping the `timeout={0}`. I explored that, but it doesn't seem to work. In my understanding, as long as we keep the `timeout={0}`, the `Transition` component instantly becomes `exited` state, which prevents us from having _exiting_ motion.